toolkit/tools/imagecustomizer/docs/building-packages.md

```diff
@@ -0,0 +1,85 @@
+# Building custom packages
+
+This is a guide on how to build your own packages for Azure Linux.
```

Comment:
I would reword this to (something around image)
`This is a guide on how to build your own packages to add a custom image in Azure Linux.`

Reply:
Fixed.